### PR TITLE
homepage

### PR DESCRIPTION
**Pull request by stavros-k**
**Description**

⚒️ Fixes  # 
image
image
image

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

File: src/components/Love.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const clickAnimationOptions = {
   animationData: clickAnimation,
   rendererSettings: {
     preserveAspectRatio: "xMidYMid slice",
+    speed: 0.5,
```
Comment (stavros-k) [inline src/components/Love.js:17]: slower animation, as it was too distracting

File: src/components/Love.js [hunk 2]
```diff
@@ -67,9 +74,8 @@ export default function Love() {
             options={clickAnimationOptions}
             isClickToPauseDisabled={true}
           />
-          <div className="overflow-hidden absolute lg:h-[400px] h-[320px] lg:w-[400px] w-[320px] bg-indigo-600/5 bottom-0 left-0 rotate-45 -z-1 rounded-3xl"></div>
+          <div className="overflow-hidden absolute lg:h-[400px] h-[400px] lg:w-[400px] w-[400px] bg-indigo-600/5 bottom-0 left-0 rotate-45 -z-1 rounded-3xl"></div>
```
Comment (stavros-k) [inline src/components/Love.js:77]: Fixes the container of the animations

File: src/css/custom.css [hunk 3]
```diff
@@ -4,11 +4,11 @@
...
+/* @font-face {
   font-family: "ADAM-CG-PRO" !important;
   src: url("/ADAM-CG-PRO.woff") format("woff") !important;
-}
-
+} */
+/*
 :root {
   --ifm-font-family-monospace: "ADAM-CG-PRO";
   --ifm-font-family-base: "ADAM-CG-PRO";
```
Comment (stavros-k) [inline src/css/custom.css:14]: removes the font as it seems its not very well viewed by many

File: src/components/Love.js [hunk 4]
```diff
@@ -50,11 +53,15 @@ export default function Love() {
...
           </Link>
-          <Icon icon="uim:discord" className="align-middle" />
-          <Link className="button button--secondary button--lg" to="https://discord.gg/tVsPTHWTtr">
-          Join Us On Discord
+          <Link
+            className="button button--secondary button--lg"
+            to="https://discord.gg/tVsPTHWTtr"
+          >
+            {/* https://icon-sets.iconify.design/akar-icons/discord-fill/ */}
+            <Icon icon="skill-icons:discord" className="align-middle m-2" />
```
Comment (stavros-k) [inline src/components/Love.js:63]: better discrod icon + inside the button

**Review by Ornias1993 — APPROVED** [review]

**Pull request merged**